assets/js/background.js

```diff
@@ -178,6 +185,103 @@ function handleOnUpdated(tabId, changeInfo, tab) {
 }
 
 /**
+ * Message listenener for context requests
+ */
+var findContext = function findContext(request, sender, sendResponse) {
+  chrome.tabs.query({
+    active: true,
+    currentWindow: true
+  }, function(tabs) {
+    var tab = tabs[0];
+    var site = getSiteFromTab(config.supportedSites, tab);
+    // Unsupported website, skip.
+    if(!site) {
+      console.warn('Unsupported');
```

Comment:
For now, it is used only from the popover (or injected), so it may be an error, but i think this should simply be removed as some of this code might be moved and reused from the background page in a near future